src/utils/utils.cairo

```diff
@@ -41,7 +41,6 @@ impl Slot0PartialEq of PartialEq<Slot0> {
         (*lhs.sqrt_price_X96) == (*rhs.sqrt_price_X96)
             && (*lhs.tick) == (*rhs.tick)
             && lhs.fee_protocol == rhs.fee_protocol
-            && lhs.unlocked == rhs.unlocked
```

Comment:
Why did we remove this?

Reply:
I removed the `unlocked` variable from here because I also removed it from the `Slot0` struct. This variable will be constantly changing, and it didn't seem right to me to have to update the entire struct just for one field. That's why I made it a storage variable. What do you think? @rcatalan98